mooncake-store/include/types.h

```diff
@@ -336,4 +351,45 @@ const static uint64_t kMinSliceSize = facebook::cachelib::Slab::kMinAllocSize;
 const static uint64_t kMaxSliceSize =
     facebook::cachelib::Slab::kSize - 16;  // should be lower than limit
 
+/**
+ * @brief Represents a contiguous memory region
+ */
+struct Segment {
+    UUID id{0, 0};
+    std::string name{};  // The name of the segment, also might be the hostname
+                         // of the server that owns the segment
+    uintptr_t base{0};
+    size_t size{0};
+    Segment() = default;
```

Comment:
Should we go ahead and remove this function?Since it's constructed, it should already be valid, right?